### PR TITLE
[chore] Default Images & Scrollable Project Modal

### PR DESCRIPTION
**Pull request by nehaahussain**
### Description
[//]: # "[Required] Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."
- Created a query to fetch the default images that correspond to the renewable energy technology from supabase.
- Made the modal scrollable.
- Added alt description based on different images.
- Changed the project image in the modal to use next/image.

### Screenshots
[//]: # "Add any screenshots that will help your PR reviewer understand what’s going on in the PR!"
[//]: # "[Required] For frontend tasks: add screenshots of the layout and expected behavior (GIFs if you’re feeling extra fancy)!"
[//]: # "Use the provided image template. Drag the desired image into the PR, then copy the link into the placeholder."

Screenshot 2024-11-03 at 6 11 18 PM

## How to review
[//]: # "/[Required] Describe the order in which to review files and what to expect when testing locally. Is there anything specifically you want feedback on? Should this be reviewed commit by commit, or all at once? What are some user flows to test? What are some edge cases to look out for?"
- press one of the markers

## Next steps
[//]: # "[Optional] - What has not yet been added to the PR, doesn’t yet have any functionality — essentially, what needs to be fixed or added."
[//]: # "[Optional] any temporary fixes in this PR that should be cleaned up later."

### Online sources
[//]: # "[Optional] insert links to any useful tutorials or documentation that you used for this PR!"

### Related PRs
[//]: # "Add related PRs you're waiting on/ PRs that will conflict, etc; if this is a refactor, feel free to add PRs that previously modified this code"

CC: @itsliterallymonique

### CODE REVIEW

File: components/ProjectModal/styles.ts [hunk 1]
```diff
@@ -34,12 +35,10 @@ export const ProjectDetails = styled.div`
   height: 100%;
   background: ${COLORS.white};
   width: 340px;
+  overflow-y: auto;
```
Comment (itsliterallymonique) [inline components/ProjectModal/styles.ts:38]: I think you also need to add `overflow-x: hidden`
Currently when I scroll horizontally on the project modal it scrolls.
Screenshot 2024-11-04 at 12 46 49 AM

**Review by itsliterallymonique — CHANGES_REQUESTED** [review]
images look good! just a quick styling fix with the horizontal scrolling. If you try to scroll horizontally the inside of the modal scrolls. So you need to add `overflow-x: hidden` somewhere in one of the modal styles / divs! There is also some lag with changing of the images -- do you know why that could be?
